<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Crear_usuario</class>
 <widget class="QWidget" name="Crear_usuario">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Cargo</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cContrasenaLineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nombreLineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nº Identificacion</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cargoLineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="contrasenaLineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelarPushButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="aceptarPushButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Aceptar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Contraseña</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirmar Contraseña</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="idLineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>E mail</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Telefono</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="emailLineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>170</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="telefonoLineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>200</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
